<launch>
  <arg name="input" default="/stereo/left/image_rect_color" />
  <arg name="debug" default="false" />
  <arg name="threshold" default="80" /> 
  <arg name="match_num" default="5" />

  <node name="linemod_no_depth" 
	pkg="linemod_recognition" type="linemod_no_depth" 
	output="screen" >
    <rosparam subst_value="true">
      template_name: $(find linemod_recognition)/templates/pie4.yml
      threshold: $(arg threshold)
      match_num: $(arg match_num)
      debug_view: $(arg debug)
    </rosparam>
    <remap from="~input" to="$(arg input)" />
  </node>
</launch>
